plumbing/object/tag.go

```diff
@@ -20,9 +20,13 @@ import (
 //
```

Comment:
Add some info about [Lightweight Tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging#_lightweight_tags) and say that this method does not return this kind, can help people that don't know this two kinds of tags.